cilium/load_balancer/docs/CiliumL4LBSetup.md

```diff
@@ -40,14 +40,14 @@ The following sections provide all the steps to prepare a setup to run Cilium L4
 ### Build eBPF-for-Windows and eBPF-for-Windows-Demo
 
 eBPF-for-Windows-Demo contains the demo agent code for Cilium load balance demo in `cilium\load_balancer\daemon`. The modified Cilium XDP code is present in a [fork](https://github.com/saxena-anurag/cilium/tree/anurag/cilium_xdp_bpf_code_changes) of the [Cilium repository](https://github.com/cilium/cilium). This project references the Cilium fork as a submodule.
-This projects also includes `eBPF-for-Windows` project as a submodule.
+This projects also includes `eBPF-for-Windows` project as a nuget package.
 
 To clone and build this project, follow the steps mentioned in the [Getting Started](../../../docs/GettingStarted.md) guide.
 
-The above guide provides steps to build both the `eBPF-for-Windows` submodule and the `eBPF-for-Windows-Demo` project.
+The above guide provides steps to build the `eBPF-for-Windows-Demo` project.
 
 Once both the projects have been built, following will be location of the build outputs:
-1. `eBPF-for-Windows`: files will be present in `external\ebpf-for-windows\x64\Release`
+1. `eBPF-for-Windows`: Nuget package will be present in `packages\eBPF-for-Windows.<version>

```

Comment:
We can remove this line, as the nuget package is not to be copied from build machine to test machine.